packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -462,27 +462,75 @@
               <div class="label-wrapper">
                 <Label>{label}</Label>
               </div>
-              <div class="attachment-field-width">
-                <KeyValueBuilder
-                  on:change={e =>
+              <div class="toggle-container">
+                <Toggle
+                  value={inputData?.meta?.useAttachmentBinding}
+                  text={"Use bindings"}
+                  size={"XS"}
+                  on:change={e => {
                     onChange(
                       {
-                        detail: e.detail.map(({ name, value }) => ({
-                          url: name,
-                          filename: value,
-                        })),
+                        detail: "",
```

Comment:
When a user is toggling from binding to attachment fields, we just clear it out as there's no real way to parse say, a javascript binding to the correct format.

Reply:
Instead of having inline functions, we could have a `clearAttachment` or similar function and to call it here. It will make it much more verbose